tests/cursorid/cursorid-var_export-001.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+MongoDB\Driver\CursorID: var_export()
```

Comment:
Small nit, but the class name is "CursorId". I realize some of the existing test files had this wrong and were using "ID". Feel free to fix that anywhere so "CursorId" is always used.

See [PHPC-1004](https://jira.mongodb.org/browse/PHPC-1004) for related discussion on this. At one point, the driver was inconsistent and used "ID" for ObjectId and "Id" for CursorId. This was never an issue in practice because class names in PHP are case-insensitive, but we standardized on "Id" for consistency.